incubator/artifactory/README.md

```diff
@@ -0,0 +1,52 @@
+# JFrog Artifactory Helm Chart
+
+## Prerequisites Details
+
+* Kubernetes 1.3 with alpha APIs enabled
+* Artifactory Pro trial license
```

Comment:
@viglesiasce how are we handling paid for software? Do we need some notice in the README, hey you need to pay for this ;)